Gifski/Gifski.swift

```diff
@@ -2,55 +2,61 @@ import Foundation
 import AVFoundation
 
 final class Gifski {
-	private(set) var isRunning = false
 	private var progress: Progress!
+	private var observation: NSKeyValueObservation?
 
 	// `progress.fractionCompleted` is KVO-compliant, but we expose this for convenience
-	var onProgress: ((_ progress: Double) -> Void)?
+	var onProgress: ((_ progress: Progress) -> Void)?
+
+	class func convertFile(
```

Comment:
No point in using `class` here as it's not going to be subclassed, so `static` would be better.

Reply:
Good point!